### PR TITLE
refactor: sort dependencies alphabetically

### PR DESCRIPTION
**Pull request by BernhardRode**
Sorted dependencies makes merging a lot easier. 
Helps a lot, when upgrading the cli and pulling in the correct deps.

### CODE REVIEW

File: packages/@angular/cli/blueprints/ng/files/package.json [hunk 1]
```diff
@@ -17,8 +17,8 @@
     "@angular/core": "<%= ng4 ? '>=4.0.0-beta <5.0.0' : '^2.4.0' %>",
     "@angular/forms": "<%= ng4 ? '>=4.0.0-beta <5.0.0' : '^2.4.0' %>",
     "@angular/http": "<%= ng4 ? '>=4.0.0-beta <5.0.0' : '^2.4.0' %>",
-    "@angular/platform-browser": "<%= ng4 ? '>=4.0.0-beta <5.0.0' : '^2.4.0' %>",
     "@angular/platform-browser-dynamic": "<%= ng4 ? '>=4.0.0-beta <5.0.0' : '^2.4.0' %>",
+    "@angular/platform-browser": "<%= ng4 ? '>=4.0.0-beta <5.0.0' : '^2.4.0' %>",
```
Comment (grizzm0) [inline packages/@angular/cli/blueprints/ng/files/package.json]: npm install will flip the order of these two. @angular/platform-browser should be first.

File: packages/@angular/cli/blueprints/ng/files/package.json [hunk 2]
```diff
@@ -32,12 +32,12 @@
...
     "jasmine-spec-reporter": "~3.2.0",
-    "karma": "~1.4.1",
     "karma-chrome-launcher": "~2.0.0",
     "karma-cli": "~1.0.1",
-    "karma-jasmine": "~1.1.0",
-    "karma-jasmine-html-reporter": "^0.2.2",
     "karma-coverage-istanbul-reporter": "^0.2.0",
+    "karma-jasmine-html-reporter": "^0.2.2",
+    "karma-jasmine": "~1.1.0",
+    "karma": "~1.4.1",
```
Comment (grizzm0) [inline packages/@angular/cli/blueprints/ng/files/package.json]: npm install will flip the order of these aswell. Shortest name first.

**Comment by BernhardRode** [comment]
I just use yarn and a sort alphabetical Plugin inside of my editor. 
 I'll double check that tomorrow.

**Comment by grizzm0** [comment]
You should have rebased, not merge master. 

And the only package I can see that is not in alphabetic order is "karma-coverage-istanbul-reporter" (comes from #4560) and it's still not in the correct place.

**Pull request closed without merging**